events-processor/models/event_test.go

```diff
@@ -37,6 +37,7 @@ func TestToEnrichedEvent(t *testing.T) {
 		assert.Equal(t, event.Source, ere.Source)
 		assert.Equal(t, 1741007009.0, ere.Timestamp)
 		assert.Equal(t, expectedTime, ere.Time)
+		assert.NotNil(t, ere.GroupedBy)
```

Comment:
Maybe check equality to ensure it's empty ?

```suggestion
		assert.Equal(t, ere.GroupedBy, map[string]string){})
```